docs/en/admins/07_LinuxUpdate.md

```diff
@@ -67,7 +67,7 @@ cd /usr/share/FreshRSS/
 
 3. Download and unzip the update file
 ```
-wget -o freshrss.zip https://github.com/FreshRSS/FreshRSS/archive/1.15.3.zip
+curl -L -o freshrss.zip https://github.com/FreshRSS/FreshRSS/archive/1.15.3.zip
```

Comment:
As an aside, the typo was introduced by @marienfressinaud when changing from master.zip to x.y.z.zip. (I have no opinion for or against wget/curl.)

Reply:
> `wget` is more usual than `curl`.
> For instance, `wget` is available by default in Ubuntu desktop, not `curl`
> http://releases.ubuntu.com/eoan/ubuntu-19.10-desktop-amd64.manifest
> So it is not a bit deal, but I would favour `wget` a bit for such a documentation

@Alkarex  - All good! I just looked it up and it's `-O` for wget, so it'll make it a *2 char* PR ;)  Just a sec here...